packages/mocks/src/getMockData.ts

```diff
@@ -54,10 +50,10 @@ export interface MockResponse {
  *
  * @property {string} pathname - The pathname of the incoming request to match against the mock handlers
  * @property {string} [method] - An optional method of the incoming request to match against the mock handlers
- * @property {PathParams} [params] - An optional object containing the path parameters of the request
+ * @property {Record<string, string | ReadonlyArray<string>>} [params] - An optional object containing the path parameters of the request
  */
 export interface MockDataRequestParameters {
   pathname: string;
   method?: string;
-  params?: PathParams;
+  params?: Record<string, string | ReadonlyArray<string>>;
```

Comment:
nice!
now we can remove the dependency on msw, I think this is the only place we use it